### PR TITLE
Nemwel blog milestone 6 : Setup views

### PR DESCRIPTION
**Pull request by Nemwel-Boniface**
## Hello reviewer(s)
In this milestone I was able to:

- [x] Implement the design from the sneak peek wireframes
- [x] Basic styling for my index and show Users views
- [x] make use of the methods I had created in the [processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md) milestone.
- [x] Check and solve all found rubocop offenses

Thank you for taking the time to review my project :pray:

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,24 @@
-<h1>Welcome to my Posts page!</h1>
\ No newline at end of file
+<div class="user">
+      <img src=<%=@user.photo%> alt="User image" height="150" width="150" >
+      <div class="user_details">
+        <h3><%= @user.name %></h3>
+        <p>Number of posts: <%=@user.post_counter%>
+      </div>
+</div>
```
Comment (Davidosky007) [inline app/views/posts/index.html.erb:7]: * After adding some data to the database by using the `rails console` [as instructed here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data) i was unable to see any post, as it is a requirement to make the views pages function as shown in the wireframes [See requirement here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design,everything%20functions%20first.). Kindly work on this 🙏

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1 +1,15 @@
...
\ No newline at end of file
+<%= stylesheet_link_tag "application"%>
+
+<div class="users">
+  <h2>List of all Users:</h2>
+
+  <%@users.each do |user|%>
+    <div class="user">
+      <img src="<%=user.photo%>" alt="User image" height="150" width="150" >
+      <div class="user_details">
```
Comment (Davidosky007) [inline app/views/users/index.html.erb:9]: * Same goes for the user view page kindly work on the

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Nemwel-Boniface 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
* It is a requirement to make your pages functions as the wireframe given in [the requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md) Kindly work on your views so that they work as in the wireframe. below is a highlighted screenshot of the requirements missing ⬇️ 
image
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nemwel-Boniface** [comment]
As requested by the previous reviewer, I will include screenshots of my project as I can see it locally:

**Frame 1 of the project description:**
image

**Frame 2 of the project description:**
image


**Frame 3 of the project description:**
image

**Frame 4 of the project description:**
image

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Nemwel-Boniface, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**